### DISCUSSION TITLE
undefined symbol: _ZN9LAMMPS_NS6LAMMPSC1EiPPci

### DISCUSSION THREAD

**Discussion by phys-chem** [ACEsuit/mace#353] [Q&A]
The compiled LAMMPS using1080Ti can run but I have to use lower torch version. I cannot run my tasks on nodes with 2080Ti. I just get this error:
`Kokkos::Cuda::initialize ERROR: running kernels compiled for compute capability 6.1 on device with compute capability 7.5 is not supported by CUDA!`
 
However, when I try compile LAMMPS using 2080Ti. I run LAMMPS and get this error even if I do not use pair mace. 
`undefined symbol: _ZN9LAMMPS_NS6LAMMPSC1EiPPci`

I am not sure what is the compiled problem. Could you give me some suggestions. Thank you very much!

**Comment by wcwitt** [comment]
Sorry I don't have much experience on that hardware. Can you post your exact install commands?

**Reply by phys-chem** [reply]
Thank you for the reply! I use this script for making and then run "make -j".
```
cmake \
    -D CMAKE_BUILD_TYPE=Release \
    -D CMAKE_INSTALL_PREFIX=$(pwd) \
    -D BUILD_MPI=yes \
    -D BUILD_OMP=yes \
    -D BUILD_SHARED_LIBS=yes \
    -D LAMMPS_EXCEPTIONS=yes \
    -D PKG_KOKKOS=yes \
    -D Kokkos_ARCH_AMDAVX=yes \
    -D Kokkos_ARCH_AMPERE100=yes \
    -D Kokkos_ENABLE_CUDA=yes \
    -D Kokkos_ENABLE_OPENMP=yes \
    -D Kokkos_ENABLE_DEBUG=no \
    -D Kokkos_ENABLE_DEBUG_BOUNDS_CHECK=no \
    -D Kokkos_ENABLE_CUDA_UVM=no \
    -D CMAKE_CXX_COMPILER=$(pwd)/../lib/kokkos/bin/nvcc_wrapper \
    -D PKG_ML-MACE=yes \
    -D CMAKE_PREFIX_PATH=$(pwd)/../../libtorch-gpu \
    ../cmake

```

**Reply by wcwitt** [reply]
What about modules (if applicable), libtorch version, etc?

**Reply by phys-chem** [reply]
torch               2.0.1+cu118
torch-ema           0.3
torchaudio          2.0.2+cu118
torchvision         0.15.2+cu118
mace-torch          0.3.4

**Reply by wcwitt** [reply]
Sorry, I'm not sure. Do you have a system administrator who can help with these kinds of things?
